Jint/Runtime/TypeConverter.cs

```diff
@@ -13,31 +13,33 @@
 
 namespace Jint.Runtime
 {
+    [Flags]
```

Comment:
What's the new requirement that makes it Flags ? Same for InternalTypes.

Reply:
Basically here ability to filter for types using enums:

https://github.com/sebastienros/jint/pull/681/files#diff-d6e42f1248b104068929cf23dbc58dddR89-R92